<?xml version="1.0"?>
<launch>

  <arg name="config_name" default="simple_robot" />
  <arg name="map_name" default="rooms_robots" />

  <node pkg="stage_ros" type="stageros" name="stage_ros" args="$(find stage_tutorial)/worlds/$(arg map_name).world" />

  <node pkg="stage_tutorial" type="reactive_navigation.py" name="reactive_controller_py" output="screen" cwd="node">
  <rosparam file="$(find stage_tutorial)/config/$(arg config_name).yaml" command="load"/>
  </node>

</launch>

